python/src/lib.rs

```diff
@@ -52,10 +53,15 @@ use crate::filesystem::FsConfig;
 use crate::schema::schema_to_pyobject;
 
 #[inline]
-fn rt() -> PyResult<tokio::runtime::Runtime> {
+fn rt_pyo3() -> PyResult<tokio::runtime::Runtime> {
     tokio::runtime::Runtime::new().map_err(|err| PyRuntimeError::new_err(err.to_string()))
 }
 
+#[inline]
+fn rt() -> &'static tokio::runtime::Runtime {
+    pyo3_asyncio::tokio::get_runtime()
+}
```

Comment:
Why don't we just create one global one in a lazy static instead? I don't think we need the `pyo3-asyncio` dependency.